internal/services/containerapps/validate/validate.go

```diff
@@ -82,21 +82,6 @@ func ContainerAppName(i interface{}, k string) (warnings []string, errors []erro
 	return
 }
 
-func ContainerCpu(i interface{}, k string) (warnings []string, errors []error) {
```

Comment:
We should still validate the CPU and memory values in the create/update for cases where users are not using a dedicated workload model, which we can discern by the absence of `workload_profile_name`.

Reply:
Hi @stephybun , this makes sense to me. But I'm wondering how we can achieve this? I've come up with some extremely hacky ways, I don't think we want it in our code base. Could you please hint me if you have some living examples? Thanks.

Reply:
I assume the hacky-ness is because of the nested expand functions? It could be possible to put the validation into a customize diff, but after chatting internally we think it is acceptable to remove the validation for now.